libopendroneid/opendroneid.h

```diff
@@ -458,6 +459,102 @@ float decodeVerticalAccuracy(ODID_Vertical_accuracy_t Accuracy);
 float decodeSpeedAccuracy(ODID_Speed_accuracy_t Accuracy);
 float decodeTimestampAccuracy(ODID_Timestamp_accuracy_t Accuracy);
 
+// OpenDroneID WiFi functions
+
+/**
+ * drone_export_gps_data - prints drone information to json style string,
+ * according to odid message specification
+ * @UAS_Data: general drone status information
+ *
+ * Returns pointer to gps_data string on success, otherwise returns NULL
+ */
+char *drone_export_gps_data(ODID_UAS_Data *UAS_Data);
+
+/**
+ * odid_message_encode_pack - encodes the messages in the odid pack
+ * @UAS_Data: general drone status information
+ * @pack: buffer space to write to
+ * @buflen: maximum length of buffer space
+ *
+ * Returns length on success, < 0 on failure. @buf only contains a valid message
+ * if the return code is >0
+ */
+int odid_message_encode_pack(ODID_UAS_Data *UAS_Data, void *pack, size_t buflen);
```

Comment:
There is already a similar function for this. I didn't look at the exact differences but would it be possible to combine this somehow?
https://github.com/opendroneid/opendroneid-core-c/blob/master/libopendroneid/opendroneid.c#L372